fili-core/src/main/java/com/yahoo/bard/webservice/sql/aggregation/DruidSqlAggregationConverter.java

```diff
@@ -0,0 +1,24 @@
+// Copyright 2017 Yahoo Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+package com.yahoo.bard.webservice.sql.aggregation;
+
+import com.yahoo.bard.webservice.druid.model.aggregation.Aggregation;
+
+import java.util.Optional;
+
+/**
+ * Provides a mapping from Druid's {@link Aggregation} to a {@link SqlAggregationBuilder}.
+ */
+public interface DruidSqlAggregationConverter {
```

Comment:
@archolewa I've done something similar to what you mentioned in #413, but tied it specifically to aggregations, Let me know if you have any concerns with this approach